packages/contracts/contracts/CyberDropBase.sol

```diff
@@ -105,7 +105,15 @@ contract CyberDropBase is CyberTokenBase {
     LibDropStorage.layout().drops[tokenId].shareCyber = _shareCyber;
     LibDropStorage.layout().drops[tokenId].creator = payable(sender);
 
+    // Effects
+    LibDropStorage.Drop storage drop = LibDropStorage.layout().drops[tokenId];
+
+    drop.minted.increment();
```

Comment:
Use `LibDropStorage.layout().drops[tokenId].minted.increment()` in place of storing local var, as it's done above, then you can remove l109